priv/repo/seeds.exs

```diff
@@ -1,2 +1,7 @@
-App.Status.upsert_status(%{text: "verified"})
-App.Status.upsert_status(%{text: "unverified"})
+App.Person.create(
+  %{
+    givenName: "Beyoncé",
```

Comment:
From my understanding this user is the one used for non loggedin user.
Renaming to "guest" would make this more obvious

Reply:
Removing the `people` schema/table will eliminate this issue. #118 🎉